demo/custom.js

```diff
@@ -245,6 +245,38 @@ shakaDemo.Custom = class {
   }
 
 
+  /**
+   * @param {!ShakaDemoAssetInfo} assetInProgress
+   * @param {!Array.<!HTMLInputElement>} inputsToCheck
+   * @return {!Element} div
+   * @private
+   */
+  makeAssetDialogContentsMediaPlaylist_(assetInProgress, inputsToCheck) {
```

Comment:
I'm concerned that this extra tab may make the dialog too crowded.  (I haven't cherry-picked this to look at the layout yet.)

Reply:
After discussing this in person, we decided to keep the separate tab but rename it to be a more general-purpose "HLS" tab.